Ops/Ops-lessons-2022/CircleCI.md

```diff
@@ -42,12 +42,60 @@ References:
 - [User](https://circleci.com/docs/2.0/glossary/#user)
 
 ### Pipelines
+Pipelines are the highest-level unit of work that includes the entire `.circleci/config.yml` file. Pipelines have workflows which coordinate jobs and they have a fixed, linear lifecycle and are associated with a specific actor. There are multiple ways to trigger a pipeline including a push to a project, a scheduled task, or manually through the CircleCI app. 
 
 ## Config.yml
 
 ### YAML Syntax
+YAML is a human-friendly data serialization standard for all programming language. It is a strict superset of JSON, which means it can do everything that JSON can and more! 
```

Comment:
```suggestion
YAML is a human-friendly data serialization standard for all programming languages. It is a strict superset of JSON, which means it can do everything that JSON can and more! 
```